src/pranitaPJun20/FibonacciSeries.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Assignment 21 : 30th Jun 2020
+Program 2: To print first n Fibonacci Numbers.
+Input n = 5
+Output = 1 1 2  3  5  8 
+
+Input: n = 10 
+Output :  1 1 2  3  5  8 13  21 34 55
+
+Hint :
+The next number is found by adding up the two numbers before it:
+the 2 is found by adding the two numbers before it (1+1),
+the 3 is found by adding the two numbers before it (1+2),
+the 5 is (2+3),
+and so on! 
+ */
+package pranitaPJun20;
+
+public class FibonacciSeries {
+//Method to find Fibonacci series for given no of terms
+	static void printFibonacciNumbers(int num1, int num2, int length) {
+		int iterationValue = 2;// iteration starts from 3rd term
+		int term1 = num1;
+		int term2 = num2;
+		System.out.print(term1 + "," + term2);// Printing values at index 0 and 1
```

Comment:
Without printing hardcoded values you can achieve expected output by making some minor changes in initialization of term1, term2 and iterationValue

Reply:
Thank you for guiding me in optimizing the code.Changes done